test/Querier.test.ts

```diff
@@ -14,25 +14,30 @@ const successQuery = async () => {
   const res = await request('/users/1');
   return res;
 };
-const storeMock: QuerierStoreType = {
-  queryKey: {
-    id: 'queryKey',
-    state: {
-      state: QuerierState.Active
-    },
-    result: null,
-    $reason: 'SomeComponent',
-    $props: {}
-  }
+
+const queryKey = 'key';
+const queryResult = users[1];
+const queryReason = 'Some Component';
+
+const createStore = (
+  state: { state: QuerierState } = { state: QuerierState.Success },
+  // tslint:disable-next-line
+  $props: any = null,
+  id: string = queryKey,
+  result: { name: string } = queryResult,
+  $reason: string = queryReason
+): QuerierStoreType => {
+  return { key: { id, result, state, $props, $reason } };
 };
+const storeMock: QuerierStoreType = createStore({ state: QuerierState.Active }, null, null);
 
-describe('Querier', () => {
+fdescribe('Querier', () => {
```

Comment:
Typo @aniaw?

Reply:
I use `fdescribe` to run one selected describe, but yes - this shouldn't be commited.

Reply:
Nice, didn't know about that :)